io/rtp/src/main/java/org/mobicents/media/server/impl/rtp/crypto/TlsUtils.java

```diff
@@ -201,14 +202,12 @@ static AsymmetricKeyParameter loadPrivateKeyResource(String resource)
 	}
 
 	static PemObject loadPemResource(String resource) throws IOException {
-		InputStream s = TlsUtils.class.getResourceAsStream(resource);
-        if (s == null) {
-            s = new FileInputStream(resource);
+        try (InputStream s = new FileInputStream(resource)) {
```

Comment:
Pay attention because the PemObject is the Reader you should be monitoring, since readPemObject throws Exception!
https://github.com/bcgit/bc-java/blob/master/core/src/main/java/org/bouncycastle/util/io/pem/PemReader.java#L26

Its also a Reader implementation, which means is Closeable. So we must use it with try-with-resources.
Also, check if the possible IOException needs to be propagated.

try(PemReader p = new PemReader(